bc_statnice/programove_vypocetni_a_informacni_systemy/01_programovaci_jazyky.md

```diff
@@ -0,0 +1,87 @@
+ 
+# 1. Programovací jazyky
+## Datové struktury programovacích jazyků
+*datová struktura* = způsob uložení informací
+- seznam
+- n-tice
+- množina
+- slovník
+- halda
+- prioritní fronta
+- fronta
+- zásobník
+- vyhledávací strom
+- červeno-černý strom
+- B-strom
+- hašovací tabulka
+
+## Řídící struktury programovacích jazyků
+**C, Java, ...**:
+if ... else, for, while, do ... while, switch
+
+**Python**:
+if...elif...else, while, for
+
+**Haskell**:
+:edit, :load, :quit, :?
```

Comment:
Opsala jsem, co bylo v sešitku, ale s tímhle nesouhlasím. Tohle jsou nějaké příkazy ghci, ipython třeba taky nějaké své má, ale řídící struktury v Haskellu jsou třeba do, if...then...else, let ... in, where, ...

Reply:
To je určitě validní připomínka a budu ráda, když to opravíš. :)